stellarphot/gui_tools/seeing_profile_functions.py

```diff
@@ -256,6 +306,29 @@ def load_fits(self):
             )
             self.exposure = np.nan
 
+    def save(self):
```

Comment:
Looks like this function/method is not tested according to codecov, I am seeing a bunch of warnings about that inline with the code review.

Reply:
About to push a fix for this....